ClemBot.Docs/docs/DesignatedChannels.md

```diff
@@ -20,6 +20,14 @@ A channel can also have multiple designations at once.
 | user_join_log  | Channel for ClemBot to log all users that join the server                             |
 | user_leave_log | Channel for ClemBot to log all users that leave the server                            |
 
+:::note
+Enabling the message_log channel causes ClemBot to store the content of messages for a period of 30 days. After which
+they are deleted. This is to enable ClemBot to notify server staff of message edits or deletions that might not be stored
```

Comment:
```suggestion
This enables ClemBot to notify server staff of message edits or deletions that might not be stored
```

Reply:
this is deleing the end of the first sentence